openshift/iib-template.yaml

```diff
@@ -156,21 +159,32 @@ objects:
           script: |
             #!/usr/bin/env bash
             #
+            set -x
+
             # adds the json request parameters to a file to be used as input data
             # for curl and preventing shell expansion.
             json_input=/tmp/$$.tmp
-            cat > $json_input <<JSON
+            json_raw_input=/tmp/$$_raw.tmp
+
+            cat > $json_raw_input <<JSON
             {
-              "fbc_fragment": $(params.fbcFragment),
+              "fbc_fragment": "$(params.fbcFragment)",
               "from_index": "$(params.fromIndex)",
               "binary_image": "$(params.binaryImage)",
-              "build_tags": $(params.buildTags),
-              "add_arches": $(params.addArches),
-              "overwrite_from_index": $(params.overwriteFromIndex),
-              "overwrite_from_index_token": "${IIB_OVERWRITE_FROM_INDEX_TOKEN}",
+              "build_tags": `echo $(params.buildTags)`,
```

Comment:
what does this do?

Reply:
The FBC endpoint has a few parameters that the builder can use. It should be a valid `json` array format. So this makes sure that the parameters will land here with the proper format in case it has some `\"`  for instance.